docs/release-notes/may-2024.md

```diff
@@ -0,0 +1,140 @@
+---
+slug: May-2024
+hide_title: true
+description: >-
+  Release notes include the changes, fixes, and additions in specific versions of Semgrep.
+toc_max_heading_level: 3
+title: May 2024
+tags:
+  - Release notes
+---
+
+# Semgrep release notes for May 2024
+
+## 🌐 Semgrep AppSec Platform
+
+### Added
+
+- **Semgrep managed scanning** is now in public beta. It enables you to scan with Semgrep on any GitHub-hosted repository without the need to change existing CI/CD configurations.
+  - Managed scanning is crucial to organizations with hundreds or thousands of repositories. Read the [<i class="fa-regular fa-file-lines"></i> docs](/deployment/managed-scanning) or the [<i class="fas fa-external-link fa-xs"></i> blog post](https://semgrep.dev/blog/2024/rapidly-deploy-code-scans-with-semgrep-managed-scanning).
+
+### Changed
+
+- Improved the onboarding experience when scanning with Semgrep on a remote repository.
+- **Projects page**: Improved the **Can't find your project?** tooltip with more troubleshooting advice.
+  - The **Sync projects** button has been moved to the **Can't find your project** tooltip.
+- Improved Jira integration and workflow. Jira tickets can now be created for Supply Chain findings, and provide better ticket summaries <!-- 14334 --> and descriptions. <!-- 14253 -->
+  - The Jira integration is in a private beta. See the [<i class="fa-regular fa-file-lines"></i> Jira documentation](/semgrep-appsec-platform/jira) to learn how it works and how to gain access to the beta.
+- **Editor and Playground**: various improvements to structure mode, including the addition of tooltips to aid in rule writing and bug fixes.
+
+## 💻 Semgrep Code
+
+### Added
+
+- **Code search** is now in beta. Code search allows you to test a Semgrep rule by running it against one or more GitHub repositories or projects instead of just a few lines of test code. Its results highlight all instances of matching code in those target repositories, allowing you to see whether your rule works as intended or not.
```

Comment:
```suggestion
- **Code search** is now in public beta. Code search allows you to test a Semgrep rule by running it against one or more GitHub repositories or projects instead of just a few lines of test code. Its results highlight all instances of matching code in those target repositories, allowing you to see whether your rule works as intended or not.
```